docs/resources/vpcep_endpoint.md

```diff
@@ -83,6 +86,12 @@ The following arguments are supported:
 
   Changing this creates a new VPC endpoint.
 
+* `routetables` - (Optional, List, ForceNew) Specifies the IDs of the route tables associated with the VPC endpoint.
+  Changing this creates a new VPC endpoint.
+
+  -> If the VPC endpoint type is gateway, the parameter is required, if the parameter is not set, will be
```

Comment:
```suggestion
  -> This field is valid only when creating a VPC endpoint for connecting a gateway VPC endpoint service. The default route table will be used when this field is not specified.
```

Reply:
Please refer to the API and modify the description of the field `enable_dns`.

Reply:
done